lib/main_common.pm

```diff
@@ -399,6 +399,11 @@ sub load_boot_tests {
     }
     if ((get_var("UEFI") || is_jeos()) && !check_var("BACKEND", "svirt")) {
         loadtest "installation/data_integrity" if data_integrity_is_applicable;
+        if ((is_arm || is_aarch64) && is_opensuse()) {
```

Comment:
I would not move this part as it is `jeos` related only and feels like it fits better in `load_jeos_tests`.